src/index.html

```diff
@@ -11,12 +11,100 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- Перша навігаційна панель -->
+    <header>
+      <div class="logo-container">
+        <a
+          class="imgLogo logo-1200"
+          href="index.html"
+        >
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Anchor links should start with the `#` symbol if they are meant to be internal page links.